tool/tsh/tsh.go

```diff
@@ -316,7 +314,6 @@ func Run(args []string) {
 	bench.Arg("[user@]host", "Remote hostname and the login to use").Required().StringVar(&cf.UserHost)
 	bench.Arg("command", "Command to execute on a remote host").Required().StringsVar(&cf.RemoteCommand)
 	bench.Flag("port", "SSH port on a remote host").Short('p').Int32Var(&cf.NodePort)
-	bench.Flag("threads", "Concurrent threads to run").Default("10").IntVar(&cf.BenchThreads)
```

Comment:
This is a backwards incompatible change (removing a flag from the CLI as well as the ability to limit the number of max requests open at a moment).  SemVer says backwards incompatible changes should only go out in Major releases (e.g. 5.0). Do we care about this regression?

Reply:
AFAIK this command is for internal use only.
It definitely shouldn't be depended on by anybody for production use.
Should be safe to remove.